app/model/User.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+
+const validateEmail = function(email) {
+    var re = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+    return re.test(email)
+};
+
+
+const UserSchema = new Schema({
+    name: {
+        type: String,
+        trim: true,
+        required: 'El nombre es obligatorio',
+        minlength: [3, 'El nombre es muy corto']
+    },
+    email: {
+        type: String, 
+        trim: true,
+        lowercase: true,
+        unique: true,
+        required: 'El email es necesario',
+        validate: [validateEmail, 'Email no valido'],
+        match: [/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/, 'Email no válido']
```

Comment:
We shouldn't use regex to validate emails. Check how people are using the "validator" package here: https://stackoverflow.com/questions/18022365/mongoose-validate-email-syntax (the 2nd answer)